pilot/cmd/pilot-discovery/main.go

```diff
@@ -95,7 +95,7 @@ func init() {
 		"Cloud Foundry config file")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.ClusterRegistriesConfigmap, "clusterRegistriesConfigMap", "",
 		"ConfigMap map for clusters config store")
-	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.ClusterRegistriesNamespace, "clusterRegistriesNamespace", "istio-system",
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.ClusterRegistriesNamespace, "clusterRegistriesNamespace", "",
```

Comment:
Why not define the default value as `istio-system` as before and set `ClusterRegistriesNamespace` to `args.Namespace` if `ClusterRegistriesNamespace` is empty?

Reply:
if the default is declared here as before, we can't tell if the user didn't set it.

Reply:
Ah, I see, thanks @tiswanso